rr_arduino/callbacks.ino

```diff
@@ -1,14 +1,14 @@
-//!ROS callbacks are automatically linked with through the Arduino IDE
-void ThrottlePIDArrayCallback(const std_msgs::Float32MultiArray & array) {
-  //!P,I,D respectively
+//ROS callbacks are automatically linked with through the Arduino IDE
+void throttle_PID_array_callback(const std_msgs::Float32MultiArray & array) {
+  // P,I,D respectively
   // for(int i = 0; i < 3; i++)
   // {
   //   //array[i] = throttle_PID_val[i];
   // }
   // ThrottlePID.SetTunings(throttle_PID_val[0], throttle_PID_val[1], throttle_PID_val[2]);
 }
 
-void cmdVelocityCallback(const std_msgs::Float32 & cmd_vel_msg)
+void cmd_velocity_callback(const std_msgs::Float32 & cmd_vel_msg)
```

Comment:
The old naming style was correct  please refer to this as a guideline https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Reply:
style corrected